### DISCUSSION TITLE
Subscription error Bad-TooManymonitoredItems

### DISCUSSION THREAD

**Discussion by fendou-jk** [eclipse-milo/milo#1278] [General]
Hi kevin，

When using the following method to monitor changes in point values, I only added 7 points. But it reported an error message of TooManymonitoriedItems. Is my subscription listening method incorrect? How can I implement a subscription to avoid this problem?

![image](https://github.com/eclipse/milo/assets/32631086/2f9798e6-6a66-4ad9-83a2-67ae933d6cc5)

![image](https://github.com/eclipse/milo/assets/32631086/e50d0acd-f9ec-4cfc-8962-25ea523a3511)

**Comment by kevinherron** [comment]
What server are you connected to?

as a client you don’t have much control over this, it’s up to the server how many MonitoredItems per Subscription, per Session, or overall per Server is too many. 7 seems low - are you sure that’s all you’re creating? Are there other subscriptions or sessions?

**Reply by fendou-jk** [reply]
Thank you for your reply. I will contact the server personnel to confirm the monitoring items information. Also, I would like to confirm if there are any issues with this subscription method in the code?
